backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/StaticFileMapper.java

```diff
@@ -40,6 +40,7 @@ public static void create(Javalin app) {
         });
 
     app.get("*/{app}/assets/<asset>", StaticFileMapper::redirectAssets);
+    app.get("*/{app}/static/<static>", StaticFileMapper::redirectStatic);
```

Comment:
this is not what I expected, need to find a way to have this more generic I suppose (artifact from the javelin migration).